QModManager/API/SMLHelper/Initializer.cs

```diff
@@ -0,0 +1,67 @@
+﻿namespace QModManager.API.SMLHelper
```

Comment:
We also need to provide an "empty" version of `SMLHelper.dll` with all calls redirected to the new API. I wonder what we will do with V1.

Reply:
I am of the opinion that this should be done as part of the final update to the original SMLHelper project/repo.  
It will be something to release along side the QMM 3.0 update with plenty of `[Obsolete]` tags indicating that this library is for backwards compatibility only and that modders should move to the one in the QMM assembly.

Reply:
Agreed, that would have to be done at the end of this pull request after everything else has been finished. About the old repository, a simple readme noting the merge could do it.

Reply:
Creating the pass-through assembly will be easiest if we start with the original master of SMLHelper and just rip out all the functionality and replace it with redirects into the QMM assembly.  

The public method calls would be easy. We would need to set up some type redirects for the Asset classes.